docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -21,6 +23,57 @@ description: >
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
 
+## [v0.19.0](https://github.com/aws/eks-anywhere/releases/tag/v0.19.0)
+
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.19.0 |    ✔    |     ✔      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     —      |    ✔    |     ✔      |  —   |
+
+### Added
+- Support for Kubernetes v1.29
+- Support for in-place Kubernetes version upgrade on Bare Metal clusters
+- Support for `etcd` scaling ([#7127](https://github.com/aws/eks-anywhere/pull/7127))
```

Comment:
Can we clarify this a bit more? "Support for `etcd` scaling with clusters with external `etcd` deployments"

(assuming this is only for external etcd clusters)